admin/app/components/solidus_admin/ui/forms/address/fields/reverse_charge_fields/component.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if Spree::Backend::Config.show_reverse_charge_fields %>
```

Comment:
I am not a huge friend of having surprising behaviour like this in a template. We could make use of [the `render?` callback](https://viewcomponent.org/api.html#render--boolean) in the component class instead.